docs/examples/movie-reviews-graph-rag.ipynb

```diff
@@ -0,0 +1,568 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # INITIAL_PROMPT_TEXT = "What are some recommendations of exiting action movies?"
</pre><p>"exiting action movies"? walking out?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/datastax/graph-rag/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>